src/content/en/2020/markup.md

```diff
@@ -1002,7 +981,7 @@ We had a look at the data and in order to better understand the results, we norm
     <thead>
       <tr>
         <th>Content attribute value</th>
-        <th>Occurrences</th>
+        <th>Pages</th>
```

Comment:
Same general problem as mentioned before and no, “pages” would be inaccurate.

Reply:
Actually this one I may be wrong. But then I feel the Pages(%) heading is inaccurate?

![image](https://user-images.githubusercontent.com/10931297/103458328-d3939200-4cfe-11eb-9779-8f6306dac694.png)

Though to be honest for this one, as there's usually only one per page it's probably the same for both!

@Tiggerito can you confirm?

Reply:
One per page and the report includes if it is missing (blank). So I say it's pages and % pages.